<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReaderGUI</class>
 <widget class="QWidget" name="ReaderGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="main_formLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="load_horizontalLayout">
     <item>
      <widget class="QLabel" name="DatasetType_label">
       <property name="text">
        <string>Dataset Type :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DatasetType_comboBox">
       <item>
        <property name="text">
         <string>Standard</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MNIST</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Load_pushButton">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="loadLabels_checkBox">
       <property name="text">
        <string>Load Labels</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="paths_verticalLayout">
     <item>
      <layout class="QGridLayout" name="pathssub_gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="ImagePath_label">
         <property name="text">
          <string>Image File :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="LoadedDataset_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelFile_label">
         <property name="text">
          <string>Label File :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labelFilepath_label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>DatasetType_comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ReaderGUI</receiver>
   <slot>dataTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Load_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ReaderGUI</receiver>
   <slot>load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dataTypeChanged(int)</slot>
  <slot>load()</slot>
 </slots>
</ui>
